src/Kanvas/Notifications/Settings/Repositories/NotificationSettingsRepository.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+namespace Kanvas\Notifications\Settings\Repositories;
+
+use Kanvas\Notifications\Settings\Models\UsersNotificationsSettings;
+
+class NotificationSettingsRepository
+{
+    /**
+     * getNotificationSettings
+     *
+     * @param  int $userId
+     * @param  int $appId
+     * @return UsersNotificationsSettings
+     */
+    public static function getNotificationSettings(int $userId, int $appId): UsersNotificationsSettings
+    {
+        return UsersNotificationsSettings::where('users_id', $userId)
+            ->where('apps_id', $appId)
+            ->get();
```

Comment:
@FredPeal dont you need a first or fail?

Reply:
nop